### ISSUE TITLE
Stack size is not unlimited on compute nodes

### ISSUE THREAD

**Issue by cedriccastagnede** [clustervision/trinityX#136]
Hello Dev,

We must add this by default into the compute osimage:

```
 echo "# ClusterVision: added to allow memlock permissions for  User space Infiniband verbs
* hard memlock unlimited
* soft memlock unlimited

# ClusterVision: added for OpenIB gen2 stack.
* - memlock unlimited

# ClusterVision: added for making sure stack size is not limited.
* - stack unlimited" > /etc/security/limits.d/91-hpc-limits.conf
```

Cheers,
Cédric

**Comment by jflf-CV** [comment]
Thanks for the information Cédric! I'll add that to the default images as soon as I have a bit of time and I'll update the ticket.

**Comment by jflf-CV** [comment]
Merged in dev branch in 46d1ab182824af7f815cc0ac4c81d42efa8b4e18, closing.
